src/layers/buildings.js

```diff
@@ -0,0 +1,58 @@
+import FeatureLayer from '@arcgis/core/layers/FeatureLayer';
+
+/**
+ * Campus buildings data layer.
+ *
+ *   - rendering is disabled due to buildings being a part of both base maps
+ *   - labels are a part of the vector base map, so they are only shown on the satellite base map
+ *
+ * @see {basemapToggle} for label show/hide logic.
+ */
+export default new FeatureLayer({
+  portalItem: {
+    id: 'c048f4a9df6344b19281876740c44ba0',
+  },
+  title: 'Buildings',
+  renderer: {
+    type: 'simple',
+    symbol: {
+      type: 'simple-fill',
+      color: [0, 0, 0, 0],
+      outline: {
+        width: 0,
+      },
+    },
+  },
+  labelsVisible: false,
+  labelingInfo: [{
+    symbol: {
+      type: 'text',
+      color: 'black',
+      haloColor: 'white',
+      haloSize: 1,
+      font: {
+        size: 8,
+      },
+    },
+    labelExpressionInfo: {
+      expression: '$feature.Building_Name',
+    },
+    minScale: 4513.988705, // vector zoom level 16
```

Comment:
Really feels like these numbers should be in some sort of constant somewhere.

Reply:
Do mean within the ArcGIS library, or somewhere we define them?

Reply:
Whoops, meant to start a review. But I was largely griping about the API. That is, it _feels_ like you shouldn't have to type in such a specific number. But apparently you do.

I guess the rationale is that "map-people" know what a 1:4500 or whatever scale looks like. That is, you don't _have_ to use an exact zoom level; maybe you just use a number that seems good based on your years of map-making experience.

Anyway, nothing to change here. If these numbers were used a lot in our code, it _might_ make sense to define a lookup of our own, but for one or two a comment is fine. Just grousing.